SConstruct

```diff
@@ -373,8 +398,19 @@ def config_env(toolchain, variant, env):
                 ])
         else:
             if toolchain == 'gcc':
+                gcc_ver = subprocess.check_output(['gcc', '-dumpversion'],
+                                                  stderr=subprocess.STDOUT).strip()
+                if gcc_ver.startswith('5'):
+                    # remove rpath and CXX11_ABI flag when distro uses
+                    # non-user installed gcc 5
+                    env.Append(CPPDEFINES={
+                        '-D_GLIBCXX_USE_CXX11_ABI' : 0
+                    })
+                    env.Append(LINKFLAGS=['-Wl,-rpath,' + gccStdLibDir()])
```

Comment:
I'm unhappy with this `rpath` setting, but I consider it better than messing with `LD_LIBRARY_PATH`. Using `gcc -v` and finding the `prefix` seems fragile. Any linux gurus have a better suggestion?  On the up side, this will go away when we use distros that ship with gcc 5 and we remove support for user built gcc 5.